ironfish/src/fileStores/config.ts

```diff
@@ -74,6 +74,14 @@ export type ConfigOptions = {
    */
   blockGraffiti: string
   nodeName: string
+  /**
+   * The number of CPU threads to us for long-running node operations, like creating
+   * transactions and verifying blocks. 0 disables threads (this is likely to cause
+   * performance issues), and -1 auto-detects based on the number of CPU cores.
+   * Each thread uses several hundred MB of memory, so try a lower value to reduce memory
+   * consumption.
+   */
+  nodeThreads: number
```

Comment:
kind of naming nit, but since this just changes how many workers we screate, can we just make this `workers`?